grammars/tree-sitter-javascript.cson

```diff
@@ -106,9 +111,13 @@ scopes:
 
   'identifier': [
     {
-      match: '^(global|module|exports|__filename|__dirname|window|document)$',
+      match: '^(global|module|exports|__filename|__dirname)$',
       scopes: 'support.variable'
     },
+    {
+      match: '^(window|event|document|performance|screen|navigator|console)$'
```

Comment:
Maybe `performance` was previously grouped with `window` and `document` because it is a browser API (as opposed to something that's available in node and other non-web environments).

I don't have strong opinions either way.

Reply:
Let's go with this. We can come back and tweak it further later.